extensions/amp-story-shopping/0.1/test/test-amp-story-shopping-config.js

```diff
@@ -200,15 +182,25 @@ describes.realWin(
     });
 
     it('does use inline config when remote src is invalid', async () => {
-      env.sandbox.stub(Services, 'xhrFor').returns({
-        fetchJson() {
-          throw new Error();
-        },
-      });
+      const exampleURL = 'invalidRemoteURL';
+      pageElement.setAttribute('src', exampleURL);
       const result = await createAmpStoryShoppingConfig('invalidRemoteUrl');
       expect(result).to.deep.eql(keyedDefaultInlineConfig);
     });
 
+    it('test invalid remote config url', async () => {
+      await createAmpStoryShoppingConfig();
+      const exampleURL = 'invalidRemoteURL';
+      pageElement.setAttribute('src', exampleURL);
```

Comment:
Similar to the questions I left above:
1) Should you remove the first `createAmpStoryShoppingConfig()` call and instead replace the `getShoppingConfig()` call below with a `createAmpStoryShoppingConfig()` call?
2) Is setting the 'src' attribute on the page element necessary even though the attachment element has the 'src' attribute set on it via `createAmpStoryShoppingConfig()`? I can't tell which element is supposed to have the 'src' attribute

Reply:
yes, that makes sense, I refactored it to make it more readable now.